e2e-tests/spring-rest-openapi-v3/src/test/kotlin/com/foo/rest/examples/spring/openapi/v3/authenticatedswaggerendpoint/AuthenticatedSwaggerEndpointController.kt

```diff
@@ -0,0 +1,38 @@
+package com.foo.rest.examples.spring.openapi.v3.authenticatedswaggerendpoint
+
+import com.foo.rest.examples.spring.openapi.v3.SpringController
+import org.evomaster.client.java.controller.AuthUtils
+import org.evomaster.client.java.controller.api.dto.auth.AuthenticationDto
+import org.evomaster.client.java.controller.problem.ProblemInfo
+import org.evomaster.client.java.controller.problem.RestProblem
+
+
+
+class AuthenticatedSwaggerEndpointController : SpringController(AuthenticatedSwaggerEndpointApplication::class.java) {
+
+
+
+    override fun getProblemInfo(): ProblemInfo {
+        return RestProblem(
+            "http://localhost:$sutPort/v3/api-docs",
```

Comment:
`"http://localhost:$sutPort/openapi-schema"` see other comment

Reply:
Hi Andrea,

Thanks for your important comment. Now I understand why we should not override the default one. I have fixed this.